BGL/include/CGAL/boost/graph/iterator.h

```diff
@@ -161,7 +161,26 @@ struct Face {
     return face(h,*g);
   }
 };
+template <typename G>
+struct OppositeFace {
```

Comment:
Browsing more in the file, I discover that the whole file is written that way. What is the rational? The naming convention you have followed looks like the one we have for concepts in the CGAL documentation.